packages/design-system/scripts/cmsds-migrate/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cmsds-migrate",
```

Comment:
In order for this to work from product repos, I think devDependency info will need to be transferred up to `packages/design-system`'s package file, and this yarn lockfile can be removed too.

Reply:
Ahh yes, ok I've moved to core and removed the lockfile.